django_guid/middleware.py

```diff
@@ -98,8 +98,9 @@ def _validate_guid(original_guid: str) -> bool:
         :return: bool
         """
         try:
-            return original_guid == uuid.UUID(original_guid, version=4).hex
+            return original_guid.replace('-','') == uuid.UUID(original_guid, version=4).hex
```

Comment:
Since it only returns a bool, I think it's slicker if we do 

```suggestion
            return bool(uuid.UUID(original_guid, version=4).hex)
```
since UUID actually validates.

Reply:
Agreeed. Ill fix the failing test and change to this implementation.